pyproject.toml

```diff
@@ -1,8 +1,10 @@
 [tool.poetry]
 name = "geoapps-utils"
 version = "0.1.0"
+license = "MIT"
 description = "Geoapps Utils"
 authors = ["Mira Geoscience <support@mirageoscience.com>"]
+homepage = "https://mirageoscience.com"
 
```

Comment:
**suggestion:** also have some classifiers (see our other modules)

Reply:
There is a separate issue just for classifiers: https://mirageoscience.atlassian.net/browse/GEOPY-1154
but I can add them on this issue if you prefer

Reply:
That's is ok then. Thanks